src/QirRuntime/README.md

```diff
@@ -11,24 +12,17 @@ This folder contains QIR runtime project, which includes implementation of the
 
 ## Build
 
+### Prerequisites
 The QirRuntime project is using CMake (3.17) + Ninja(1.10.0) + Clang++(11.0.0). Other versions of the tools might work
- but haven't been tested. Only x64 architecture is supported.
+ but haven't been tested. Only x64 architecture is supported.  
+For runing the PowerShell scripts below use the powershell core (`pwsh`),
```

Comment:
Note that from version 7.0, "PowerShell Core" is just "PowerShell." (Similar to that .NET 5 is just ".NET" even though it's the next version of .NET Core.)

```suggestion
For runing the PowerShell scripts below use PowerShell Core or PowerShell (`pwsh`),
```